docs/README.md

```diff
@@ -51,10 +51,11 @@ use the following notation:
 ```bash
 # Example code::
```

Comment:
Why is there a extra : in "Example Code"? Just curious.

Reply:
Not sure. However, this is not part of the PR 😅